CHANGELOG.md

```diff
@@ -15,10 +15,27 @@ and includes an additional section for migration notes.
 
 ## [Unreleased]
```

Comment:
Add the following sections under Unreleased so it is available for the next release

### Added
### Changed
### Deprecated
### Removed
### Fixed
### Security
### Migration Notes

Reply:
Added.